scripts/run-tests.sh

```diff
@@ -17,11 +17,11 @@ if [[ -f ./unittest_targets ]] ; then
   for utest in $(cat ./unittest_targets) ; do
     echo "INFO: Starting unit tests for target $utest"
     logfilename=$( echo $utest |  cut -f1 -d ':' | rev | cut -f1 -d'/' | rev )
-    if ! $scriptdir/run-tests.py -j $JOBS $utest  &> /root/contrail/logs/$logfilename ; then
+    if ! $scriptdir/run-tests.py -j $JOBS $utest  &> /root/contrail/ut_logs/$logfilename ; then
```

Comment:
please define path '/root/contrail/ut_logs/' once above. like:
logs_path=/root/contrail/logs/

and use it code